### DISCUSSION TITLE
iOS Notification Error for Detection

### DISCUSSION THREAD

**Discussion by templeowls21** [blakeblackshear/frigate#14391] [Ask A Question]
I'm using a CloudFlare tunnel to access HA when not on my network. I'm using this [Blueprint](https://github.com/SgtBatten/HA_blueprints/) to send me notifications when there's motion. It works great when I'm on my home network but when connected via the CF tunnel, the screenshot doesn't load and it sends me to an error page when I try to access the clip.

The url for the clip is http://<cloud-flare-url>/api/frigate/notifications/1729092368.733254-8vq2k3/driveway/clip.mp4?external_auth=1

Hoping there's a way for it to account for the CloudFlare tunnel if possible. Any suggestions? I'm not sure how to tackle

**Comment by NickM-27** [comment]
What is the base url you are using? Base url must be used to access remotely

**Reply by templeowls21** [reply]
Sorry, the url for the clip is: http://**_personal-cloud-flare-url_**/api/frigate/notifications/1729092368.733254-8vq2k3/driveway/clip.mp4?external_auth=1

I'd rather not share my cloudflare url if thats possible tbh

**Reply by NickM-27** [reply]
and the url points to home assistant, correct?

**Reply by templeowls21** [reply]
It's a CloudFlare tunnel for me to assess HA when not on my home network

**Reply by NickM-27** [reply]
right, just making sure it doesn't point to frigate. I have this same setup and it works fine so something must be wrong with the url or the way the integration is configured

**Reply by templeowls21** [reply]
Gotcha. I'm guessing it's probably the Blueprint, right? Would you mind sharing your code for notifications?

**Reply by NickM-27** [reply]
well, these days I just use the built in notifications coming in the next version of frigate. But before that I used https://github.com/blakeblackshear/frigate/discussions/11554

**Reply by templeowls21** [reply]
Gotcha, Thank you!! When is the next version going live?

**Reply by NickM-27** [reply]
There is no set timeframe, when it is ready

**Comment by tondef** [comment]
I think the issue is that Apple devices support H.265, but they require that the file be tagged properly. Specifically, the video file must be tagged as hvc1. In ffmpeg, that is done with: -tag:v hvc1 

So one way to accomplish this in Frigate is to, in your frigate.yaml file, apply that tag as the video is inputted: 

ffmpeg:
  input_args: -tag:v hvc1 

Try that and see if new notification clips are viewable in iOS and MacOS.

**Reply by NickM-27** [reply]
this should not be an input arg, but rather an output arg
